iceoryx_examples/callbacks_in_c/README.md

```diff
@@ -126,8 +136,11 @@ iox_listener_detach_subscriber_event(listener, subscriberRight, SubscriberEvent_
 
 In a last step we have to release all acquired resources
 
+<!--[geoffrey][iceoryx_examples/callbacks_in_c/ice_c_callbacks_subscriber.c][cleanup]-->
 ```c
+#if !defined(_WIN32)
 pthread_join(heartbeatTriggerThread, NULL);
+#endif
```

Comment:
I wonder whether this 3 lines could be removed without reducing the information for the user.

Reply:
I asked myself the same question :smiley: